acceptance/bundle/variables/git-branch/databricks.yml

```diff
@@ -0,0 +1,17 @@
+bundle:
+  name: git
+  git:
+    # This is currently not supported
+    branch: ${var.deployment_branch}
+
+variables:
+  deployment_branch:
+    # read from git; inferred should be set to true; current it's not happening
```

Comment:
Does this belong to the stanza above? I don't understand what you expect to happen.

Reply:
Updated the description -- https://github.com/databricks/cli/pull/2118/commits/fe0ec6b5e41e600c2a09bf4b3a2f939b391b4e80